File: atcoder/abc044/abc044_a.py
def cost(N, K, X, Y):
    cost = 0
    for i in range(1,N+1):
        if i <= K:
            cost += X
        else:
            cost += Y
    return cost

N = int(input())
K = int(input())
X = int(input())
Y = int(input())

print(cost(N, K, X, Y))

File: atcoder/abc044/abc044_a.cpp
#include <iostream>

using namespace std;

int cost(int N, int K, int X, int Y){
    int cost = 0;
    for (int i = 1; i <= N; i++){
        if(i <= K){
            cost += X;
        }else{
            cost += Y;
        }
    }
    return cost;
}


int main()
{
    int N, K, X, Y;
    cin >> N;
    cin >> K;
    cin >> X;
    cin >> Y;
    
    cout <<  cost(N, K, X, Y) << endl;

    return 0;
}

